scripts/fix-mjs.sh

```diff
@@ -1,6 +1,28 @@
+#!/bin/bash
+
+echo "Updating .js files..."
 for file in ./dist/esm/*.js; do
   echo "Updating $file contents..."
-  sed -i '' "s/\.js'/\.mjs'/g" "$file"
+  # Use in-place edit correctly for both Linux and macOS
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    sed -i '' "s/\.js'/\.mjs'/g" "$file" # macOS
```

Comment:
As a separate issue, I realized this doesn't work if your linter is set to use regular quotes (`"`) instead of single quotes (`'`) because in that case `.js'` won't match (since it will be `.js"`

This is not an issue introduced in this PR (it's an issue that has existed from the start) so it shouldn't block merging this PR, but it would be nice to fix this as well so either quote type works